### PR TITLE
Move MvRx to AndroidX

### PR DESCRIPTION
**Pull request by rossbacher**
- Update all dependencies and usages to AndroidX
- Update to AGP 3.2 and Kotlin 1.2.41

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -1,21 +1,26 @@
 buildscript {
-    ext.kotlinVersion = '1.2.41'
-    ext.supportLibVersion = '27.1.1'
-    ext.lifecycleVersion = '1.1.1'
+    ext.kotlinVersion = '1.2.51'
+    ext.androidxVersion = '1.0.0'
```
Comment (bernaferrari) [inline build.gradle]: While this works, they are being updated independently (there is already, for example, androix.appcompat 1.0.2, which fixes important issues with PrecomputedTextCompat, while RecyclerView is still at 1.0.0)

Comment (rossbacher) [reply build.gradle]: Good point and I wanted to break this outer later. I did not realize that an independent bugfix release already happened. Let me update that.

**Comment by gpeal** [comment]
@rossbacher Let's hold off on merging this until we do one final release with any changes we want right before it.

**Comment by rossbacher** [comment]
Replaced by https://github.com/airbnb/MvRx/pull/194

**Pull request closed without merging**